src/containers/NewInvoicePage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Immutable from 'immutable';
+import { connect } from 'react-redux';
+import { createInvoice } from '../thunks';
+import  CreateInvoiceForm from '../components/CreateInvoiceForm';
+
+function NewInvoicePage({clients, users, onCreateClick}) {
+  return (
+    <CreateInvoiceForm onSubmit={onCreateClick}
+                         clients={clients}
+                         users={users}/>
+  );
+}
+
+function clientFromProject(project) {
+  const name = project.get('name');
+  return name.split('|', 2)[0];
```

Comment:
This is an artefact of importing project names from harvest? If it's not a time-tracker specific convention, I think this should be extracted out and labelled as a harvest import artefact.

If it _is_ a time-tracker specific convention, why? Could use a better data structure instead of string munging no?